HF-122: Framework integration guides for React, Angular, Vue, Svelte (#1653)
handsontable/hyperformula: docs/guide/integration-with-vue.md

# Integration with Vue

> The HyperFormula API is identical in a Vue 3 app and in plain JavaScript. What changes is how you keep the engine out of Vue's reactivity system and how you surface its values into the template.

Reviewer: ```suggestion
The HyperFormula API is identical in a Vue 3 app and in plain JavaScript. This guide demonstrates how HyperFormula integrates with the Vue reactivity system and how to surface its values in the template.
```

Author: Adopted verbatim in [`b080681`](https://github.com/handsontable/hyperformula/pull/1653/commits/b080681dc). Better wording — thanks.
